serialzy/serializers/exception.py

```diff
@@ -0,0 +1,63 @@
+import logging
+import sys
+from inspect import isclass
+from typing import Any, BinaryIO, Callable, Dict, Type, Union, Optional
+
+from packaging import version  # type: ignore
+from serialzy.api import Schema, VersionBoundary
+from serialzy.base import DefaultSchemaSerializerByValue
+from serialzy.utils import cached_installed_packages
+
+_LOG = logging.getLogger(__name__)
+
+
+class ExceptionSerializer(DefaultSchemaSerializerByValue):
+    DATA_FORMAT = "raw_exception"
+
+    def _serialize(self, obj: BaseException, dest: BinaryIO) -> None:
+        from tblib import pickling_support  # type: ignore
```

Comment:
What does tblib actually do?

Reply:
Refactored to use just pickling tracebacks